src/oc_erchef/apps/oc_chef_wm/itest/mocks/chef_otto.erl

```diff
@@ -6,12 +6,9 @@
 -module(chef_otto).
 
 -export([
-         connect/0,
-         fetch_org_id/2
+         connect/0
```

Comment:
In chef-mover that shouldn't be an issue since we lock the versions of all the dependency.  But I'm not sure if there is other upgrade machinery that would require it.  I can't think of any.